swagger.yaml

```diff
@@ -0,0 +1,666 @@
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/PagePayload'
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Invalid request
+        '401':
+          description: Invalid Authorization Header
+        '402':
+          description: Non Identifiable Request
+        '403':
+          description: Request Body Read Failed
+        '413':
+          description: Request size too large
+        '429':
+          description: Too many requests
+        '498':
+          description: Missing Authorization Header
+  /v1/screen:
+    post:
+      tags:
+        - HTTP API
+      summary: Screen
+      description: >-
+        The screen call is the mobile equivalent of the page call. It lets you
+        record whenever your user views their mobile screen with any additional
+        relevant information about the screen.
+      operationId: Screen
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/ScreenPayload'
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Invalid request
+        '401':
+          description: Invalid Authorization Header
+        '402':
+          description: Non Identifiable Request
+        '403':
+          description: Request Body Read Failed
+        '413':
+          description: Request size too large
+        '429':
+          description: Too many requests
+        '498':
+          description: Missing Authorization Header
+  /v1/group:
+    post:
+      tags:
+        - HTTP API
+      summary: Group
+      description: >-
+        The group call lets you link an identified user with a group such as a
+        company, organization, or an account. It also lets you record any custom
+        traits associated with that group, like the name of the company, the
+        number of employees, etc.
+      operationId: Group
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/GroupPayload'
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Invalid request
+        '401':
+          description: Invalid Authorization Header
+        '402':
+          description: Non Identifiable Request
+        '403':
+          description: Request Body Read Failed
+        '413':
+          description: Request size too large
+        '429':
+          description: Too many requests
+        '498':
+          description: Missing Authorization Header
```

Comment:
Don't think we send these response codes: 402, 403, 498. Can u please check once? This is the [status map](https://github.com/rudderlabs/rudder-server/blob/bd365b50d7b537c5144cee495af6f491ed2736c1/gateway/response/response.go#L74) that we use while sending responses to the client.

Reply:
403 is present in the status map, will remove 402 and 498.